docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.7'
+
+services:
+  postgres:
+    image: postgres
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=${POSTGRES_DB}
+
+  dbt:
+    image: medicmobile/dataemon:latest
+    build: .
+    working_dir: /dbt/
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
```

Comment:
It would make it more clear what variables are needed to successfully start this service if these env variables as included in the .env file